_includes/methods/method.html

```diff
@@ -0,0 +1,46 @@
+<span class="anchor" id="{{ this_method.title | slugify }}"></span>
+<article class="method" itemscope itemtype="http://schema.org/Article">
+  <div class="method--panel method--panel--front">
+    <h1 class="method--title" itemprop="headline"><a href="{{ site.baseurl }}{{ method.url }}">
+      {% if this_method.title %}
```

Comment:
[question] Are there cases where the methods don't have a title but they do have what / why / etc?

Reply:
I'm not sure, but I'm presuming there are since this conditional existed in the old template. I tried to port over the old template as much as I could because it's hard to know those things until all the content is ported.